docs/src/developers_guide/contributing_documentation_full.rst

```diff
@@ -147,7 +147,7 @@ can exclude the module from the API documentation.  Add the entry to the
 Gallery
 ~~~~~~~
 
-The Iris :ref:`sphx_glr_generated_gallery` uses a sphinx extension named
+The Iris :ref:`gallery_index` uses a sphinx extension named
```

Comment:
`sphinx-gallery` 0.11.x doesn't appear to automagically create the `sphx_glr_generated_gallery` namespace reference, so defaulting back to the static reference of `gallery_index`, which is guaranteed to always be available.